packages/components/src/VirtualizedList/CellLabel/CellLabel.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import CellLabel from './CellLabel.component';
+
+describe('CellLabel', () => {
+	it('should render label', () => {
```

Comment:
WDYT to add a UT with a mount and have a snapshot of the HTML ?

Reply:
The test was here to ensure that we really have the right classes, because they do 100% of the job.